docs/src/release_notes.rst

```diff
@@ -28,6 +41,14 @@ Upcoming
     walletPayments     0.1.9
     ================== =======
 
+    Additionally, addressing CC-1 requires updating the ``DsoRules`` configuration to enforce
+    a vote cooldown time of 1 minute by setting:
+
+    .. code-block:: none
+
+      voteCooldownTime.microseconds = 60000000
```

Comment:
@moritzkiefer-da : I haven't yet validated that this is the exact way how a reltime value is displayed. If you know by heart that this is right, then I'll happily take your word for it. Otherwise, I'll start a local frontend instance to double-check.